web/content/js/versions.js

```diff
@@ -199,6 +199,206 @@ const navVersions = [
       }
     ]
   },
+  {
+    "title": "Foreman 3.10 - Katello 4.12 (RC)",
+    "path": "3.10",
+    "builds": [
+      {
+        "title": "Foreman on EL",
+        "filename": "index-foreman-el.html",
+        "guides": [
```

Comment:
Many of these guides aren't ready and hidden on stable branches. I'd look at 3.9 and see which ones should be hidden. Same for Foreman on Debian.

Reply:
Well, the docs are "special". Many guides aren't really ready for non-Katello. Ideally speaking we'd review all of them and fix incorrect content, but until then we hide a log of guides for stable releases.